### PR TITLE
Add html css and javascript file

### PR DESCRIPTION
**Pull request by Pazzo97**
This Pull Request is about the Nobilis Summer BootCamp for web development.
Files that I have added:

1. index.html file for html
2. style.css file for css
3. script.js file for javascript
4. RADME.md file for project description
5. images folder
6. Html-CSS-javascript linters configuration files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+
+Feel free to check the [issues page](https://github.com/Pazzo97/Capstone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Credit to  Cindy Shin (https://www.behance.net/adagio07) who designed this website
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly rephrase this as `Credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).` as the credit should provide all the links in the Readme.

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+- Html5
+- CSS3
+- Javascript
+
+## Live Demo
+
+[Live Demo Link](https://loom.com/share/44df9d95b61c462e8fc4c35a21c0c8b0)
+
+
```
Comment (sja-thedude) [inline README.md:16]: - [ ] Kindly add a live link to the project and use GitHub Pages to deploy the web pages.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+  <div class="homepage">
+
+    <header class="header">
+
+      <nav>
```
Comment (sja-thedude) [inline about.html:24]: - [ ] Kindly add the navbar at the top which is highlighted in the image given below as per the design given by Cindy shin for both the home page and the about page.
Screenshot 2022-04-13 at 11 12 29 PM

File: about.html [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+    <script src="./script.js"></script>
+
+  </div>
+
+  <footer>
+
+    <div class="footer-content">
+
+      <h3>Follow us</h3>
```
Comment (sja-thedude) [inline about.html]: - [ ] Kindly add and align the logo to the left and text align to the right as per the design in the footer. Make sure to remove social media links and follow the design given.

Original design:
Screenshot 2022-04-13 at 11 27 36 PM

Your design:
Screenshot 2022-04-13 at 11 24 11 PM

File: script.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor',
+  },
+
+  {
+    image: 'images/Passport-Picture.JPG',
+    name: 'Patrick Mukunzi',
+    title: 'Lecturer Texas Tech',
+    bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor',
+  },
+
```
Comment (sja-thedude) [inline script.js:58]: - [ ] Kindly add 2 more featured lecturers as per the design.
- [ ] Also make sure to add the corner image displayed at the top left for all images in the speaker section.
Screenshot 2022-04-14 at 12 01 57 AM

File: homepage.html [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+      <ul class="socials">
+        <li><a href="#"><img src="https://img.icons8.com/color/48/000000/facebook-new.png" alt="facebook" /></a></li>
+        <li><a href="#"><img src="https://img.icons8.com/fluency/48/000000/instagram-new.png" alt="instagram" /></a>
+        </li>
+        <li><a href="#"><img src="https://img.icons8.com/color/48/000000/twitter--v1.png" alt="twitter" /></a></li>
+        <li><a href="#"><img src="https://img.icons8.com/glyph-neue/64/000000/github.png" alt="github" /></a></li>
+        <li><a href="#"><img src="https://img.icons8.com/color/48/000000/linkedin.png" alt="linkedin" /></a></li>
+      </ul>
+
```
Comment (sja-thedude) [inline homepage.html]: - [ ] Kindly add footer for both the homepage and about page as per the design. Currently, the footer is not added for the mobile version. Fix it.
Screenshot 2022-04-14 at 12 09 55 AM

File: homepage.html [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+          <li><a href="about.html">About Us</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Lecturers</a></li>
+          <li class="closeMenu"><img src="https://img.icons8.com/ios-glyphs/30/000000/delete-sign.png"
+              alt="close menu" /></li>
+        </ul>
+
+      </nav>
+
+      <div class="bootcamp-headings">
```
Comment (sja-thedude) [inline homepage.html]: - [ ] Kindly use semantic tags, instead of div use `section` for best HTML practice.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,884 @@
+.about-file {
+  background-image: url('images/About.svg');
+}
+
+@media only screen and (max-width: 768px) {
```
Comment (sja-thedude) [inline style.css:1]: - [ ] Kindly make sure that media queries are used to change the design from mobile to desktop. Kindly use media queries only for the desktop version, not the mobile one.

File: homepage.html [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+
+          <div class="card-box">
+
+            <div class="program-icon"><img src="https://img.icons8.com/dusk/64/000000/css3.png" alt="css3" /></div>
+
+            <div class="program-title">
+              <h3>CSS3</h3>
+            </div>
+
+            <div class="program-description">
```
Comment (sja-thedude) [inline homepage.html]: - [ ] Kindly align the items as per the original design given. Follow the spacing and colours as per the design.
- [ ] Kindly follow the correct colour for the background of all the cards of the main program.
- [ ] Make sure to add the correct colour and a hover effect to the button as per the design.

Original Mobile version:-
Screenshot 2022-04-14 at 12 37 20 AM
Original Desktop Version-:
Screenshot 2022-04-14 at 12 32 33 AM

Your desktop design-:
Screenshot 2022-04-14 at 12 32 49 AM
Your mobile design-:
Screenshot 2022-04-14 at 12 27 10 AM

File: script.js [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+];
```
Comment (sja-thedude) [inline script.js:59]: - [ ] Kindly align the items as per the original design given For the mobile version.

Your design:
Screenshot 2022-04-14 at 12 27 49 AM

Original design:
Screenshot 2022-04-14 at 12 42 11 AM

File: homepage.html [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+  </header>
+
+  <main>
+
+    <div class="main-program">
+
+      <div class="main-program-contents">
+
+        <div>
+          <h2 class="main-program-head">Main program</h2>
```
Comment (sja-thedude) [inline homepage.html]: - [ ] Kindly follow the correct font family for Main Program as per the original design.
Your design:
Screenshot 2022-04-14 at 12 26 52 AM

Original design:
Screenshot 2022-04-14 at 12 47 11 AM

File: homepage.html [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+    <img class="header-img" src="images/pexels-helena-lopes-933964.jpg" alt="cover image">
+
+    <div class="header-container">
+
+      <nav>
+        <div class="logo"><img src="images/logo.jpg" alt="logo">
+          <p>Nobilis</p>
+        </div>
+        <div class="openMenu"><img src="https://img.icons8.com/ios-filled/50/000000/menu--v1.png" alt="open menu" />
+        </div>
```
Comment (sja-thedude) [inline homepage.html]: - [ ] KIndly make sure that the mobile menu has white background.

- Original design: 
Screenshot 2022-04-13 at 11 30 49 PM

- Your design:
Screenshot 2022-04-13 at 11 44 36 PM

File: about.html [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+      </div>
+
+      <div class="summary-about">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eveniet consequatur assumenda officia, nostrum,
+          distinctio laudantium omnis cumque officiis doloribus reiciendis cupiditate ipsa. Laborum illo iusto eius,
+          sunt veniam impedit! Eligendi praesentium aperiam voluptatem facere cum?</p>
+      </div>
+
+      <div class="contact">
+        <p>Please contact us for any further questions <br> about Nobilis summer bootcamp</p>
```
Comment (sja-thedude) [inline about.html]: - [ ] Kindly align the items as per the design given. Make sure the text is centered with correct spacing. Also, make sure to follow the correct text colour.

Original design (mobile):
Screenshot 2022-04-14 at 12 55 30 AM

Original design (desktop):
Screenshot 2022-04-14 at 12 30 28 AM

Your design (mobile):
Screenshot 2022-04-14 at 12 28 33 AM

Your design (desktop):
Screenshot 2022-04-14 at 12 30 56 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @Pazzo97, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
- [ ] Good PR description 👍 
- [ ] Good Video presentation 👍 

## Suggested changes
- [ ] Kindly make sure to add a summary of the project in the readme file.
- [ ] Make sure to follow the spacing colour and font for the entire project.
- [ ] _Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 14]
```diff
@@ -1 +1,56 @@
...
+## Built Withhttps:
+
+- Html5
+- CSS3
+- Javascript
+
+## Live Demo
+
+[Live Demo Link](https://pazzo97.github.io/Capstone/)
```
Comment (OLIPLICHE) [inline README.md:14]: - [ ] Great Job for adding the live link as requested by the previous review, Please make sure the project is deployed properly, and all pages, links, and images work as expected.

File: about.html [hunk 15]
```diff
@@ -0,0 +1,179 @@
...
+  </div>
+
+  <script src="./script.js"></script>
+
+  <footer>
+
+    <div class="footer-content">
+
+      <h3>Follow us</h3>
+
```
Comment (OLIPLICHE) [inline about.html:165]: - [ ] Please follow the requirement from the previous review: "Make sure to remove social media links and follow the design given."
- [ ] Original Design

b1

- [ ] Your Design



b2

File: homepage.html (hunk 7 above)
Comment (OLIPLICHE) [reply homepage.html]: - [ ] Please use semantic tags: sections use `section`, articles the nav and footer use `nav` and `footer`. Use `<section>` instead of `<div>`. Rember what the previous review said

File: homepage.html [hunk 16]
```diff
@@ -0,0 +1,250 @@
...
+
+        <ul class="navSocials">
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/facebook-new.png" alt="facebook" /></a>
+          </li>
+          <li><a href="#"><img src="https://img.icons8.com/fluency/48/000000/instagram-new.png" alt="instagram" /></a>
+          </li>
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/twitter--v1.png" alt="twitter" /></a></li>
+          <li><a href="#"><img src="https://img.icons8.com/glyph-neue/64/000000/github.png" alt="github" /></a></li>
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/linkedin.png" alt="linkedin" /></a></li>
+        </ul>
```
Comment (OLIPLICHE) [inline homepage.html]: - [ ] Please follow the instructions and fix what the previous review said: "The top short bar section with social media links, logout, and language links aligned to the right"
- Original Design
b4
- Your Design


b5

File: about.html [hunk 17]
```diff
@@ -0,0 +1,179 @@
...
+          <div class="openMenu"><img src="https://img.icons8.com/ios-filled/50/000000/menu--v1.png" alt="open menu" />
+          </div>
+
+          <ul class="mainMenu">
+            <li><a href="about.html">About Us</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Lecturers</a></li>
+            <li class="closeMenu"><img src="https://img.icons8.com/ios-glyphs/30/000000/delete-sign.png"
+                alt="close menu" /></li>
+          </ul>
```
Comment (OLIPLICHE) [inline about.html]: - [ ] Please make sure It's possible to navigate between the pages, there's a link from `Home` page to `About` page and the opposite too.

- Your Design



b6

File: homepage.html (hunk 6 above)
Comment (OLIPLICHE) [reply homepage.html]: - [ ] Great Job on implementing the change. Make sure that the background color of the footer on About page is `black` and the one on the home page is `white`.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi, @Pazzo97 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+    bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor',
+  },
+
+  {
+    image: 'images/Passport-Picture.JPG',
+    name: 'Patrick Mukunzi',
+    title: 'Faculty head EMU',
+    bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor',
+  },
+
```
Comment (aimemalaika) [inline script.js:44]: - [ ] Kindly make sure to add the corner image displayed at the top left for all images in the speaker section.163269807-49f1a34c-6c90-4a1b-b4a5-45d862a74061

File: index.html [hunk 19]
```diff
@@ -0,0 +1,265 @@
...
+
+              <div class="program-icon"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png" alt="html5" />
+              </div>
+
+              <div class="program-title">
+                <h3>Html5</h3>
+              </div>
+
+              <div class="program-description">
+                <p>HTML5 is a markup language used for structuring and presenting content on the World Wide Web. It is
```
Comment (aimemalaika) [inline index.html:121]: - [ ] As requested by the previous reviewer kindly follow the correct font family for Main Program as per the original design.
Your design:
Screenshot 2022-04-14 at 12 26 52 AM

Original design:
Screenshot 2022-04-14 at 12 47 11 AM

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Pazzo97, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
the-simpsons-homer-simpson

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 20]
```diff
@@ -0,0 +1,265 @@
...
+
+          <div>
+            <h2 class="main-program-head">Main program</h2>
+          </div>
+
+          <div>
+            <hr class="line">
+          </div>
+
+          <div class="card">
```
Comment (Donard97) [inline index.html:109]: - [ ] Well done on implementing the other requested changes from the other reviewers but as they requested you to change to the background of the main program section and kindly fix the design of the cards for the mobile screen to stick as much as you can to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Keep rocking 💪 

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

File: script.js [hunk 21]
```diff
@@ -0,0 +1,84 @@
...
+}
+function close() {
+  mainMenu.style.top = '-120%';
+}
+
+openMenu.addEventListener('click', show);
+closeMenu.addEventListener('click', close);
+
+const featureLecturersList = [
+  {
```
Comment (Donard97) [inline script.js:17]: - [ ] Kindly change the design of the cards as the previous reviewer asked you to change to stick as much as you can to the design so please change it. Keep coding 💪 

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi  @Pazzo97, 👋  
    
 _Great work on making the changes requested by a previous reviewer 👏 💯_ 

**You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 💪** 
    
giphy

## Required Changes ♻️

- [ ] Please check the comments under the review.
    
## Optional suggestions
    
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
    
**Cheers and Happy coding!👏👏👏**
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
------
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Donard97 — APPROVED** [review]
## Hello @Pazzo97 , 👋  

**Your project is complete!  There is nothing else to say other than... it's time to merge it 👏 💯** 

## Congratulations! 🎉 🎉 🎉 

giphy (13)

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
